plugins/versionpress/tests/GitRepositoryTests/MergeDriverTest.php

```diff
@@ -60,6 +61,43 @@ public function mergeDriverUninstalledCorrectly() {
     /**
      * @test
      */
+    public function gitAttributesRemovedWhenEmpty() {
+        $this->installMergeDriver('auto');
+        MergeDriverInstaller::uninstallMergeDriver(self::$repositoryDir, __DIR__ . '/../..', self::$repositoryDir);
+        $this->assertFileNotExists(self::$repositoryDir . "/.gitattributes");
+    }
+
+    /**
+     * @test
+     */
+    public function gitAttributesIsSameAfterInstallAndUninstall() {
+        $gitAttributesContent = "*.txt text\n*.jpg binary";
+        file_put_contents(self::$repositoryDir . "/.gitattributes", $gitAttributesContent);
+        $this->installMergeDriver('auto');
+        MergeDriverInstaller::uninstallMergeDriver(self::$repositoryDir, __DIR__ . '/../..', self::$repositoryDir);
+        $this->assertEquals($gitAttributesContent, file_get_contents(self::$repositoryDir . "/.gitattributes"));
+    }
+
+    /**
+     * @test
+     */
+    public function mergeDriverIsNotAddedWhenPresent() {
+        $gitattributesContents = file_get_contents(__DIR__ . '/../../src/Initialization/.gitattributes.tpl');
+        $gitConfigContents = "[merge \"vp-ini\"]";
+        $gitattributesVariables = array(
+            'vpdb-dir' => rtrim(ltrim(PathUtils::getRelativePath(self::$repositoryDir, self::$repositoryDir), '.'), '/\\')
```

Comment:
Anyway, relative path from `self::$repositoryDir` to `self::$repositoryDir` is an empty string.